packages/manager/modules/pci/src/components/project/storages/databases/queryStatistics.class.js

```diff
@@ -0,0 +1,45 @@
+export default class QueryStatistics {
+  constructor({
```

Comment:
Nitpick: can you accept individual arguments in constructor insteadof object like
constructor(query,
      rows,
      calls,
      minTime,
      maxTime,
      meanTime,
      stddevTime,
      totalTime) { }
Doing this will be simple to understand and the caller can pass postgress/mysql objects. And its callers responsibility to map them.

Reply:
Doesn't that mean de structuring object inside the service API response ?

Reply:
no need to de-structure the object. All you have to do is pass the right parameters while creating QueryStatistics object. ex:
const mysqlQS = new QueryStatistics( query,
    rows,
    calls,
    minTime,
    maxTime,
    meanTime,
    stddevTime,
    totalTime);

const postgressQS = new QueryStatistics( digestText,
    sumRowsSent,
    countStar,
    minTimerWait,
    maxTimerWait,
    avgTimerWait,
    sumTimerWait);